src/runtime/mprof.go

```diff
@@ -0,0 +1,6 @@
+package runtime
+
+// Stack is a stub, not implemented
+func Stack(buf []byte, all bool) int {
```

Comment:
You could add this to stack.go, which has similar functions (to avoid yet another small file).

None of these are likely to be implemented considering what TinyGo is designed for.